src/components/organisms/LeftSection/LeftSection.module.css

```diff
@@ -0,0 +1,30 @@
+.searchSection {
+  height: 80px;
+}
+
+.searchChatField {
+  width: 100%;
+  border: #3f51b5 solid 1px !important;
```

Comment:
Agnieszko, co tu ten `!important` robi? no nie możemy tak zostawić. Jest parę rozwiązań, ale to możemy się jutro w biuro spotkać i to ogarnąć. "!important" nie może zostać :(

Reply:
Może wykorzystanie metodologi BEM pomoże Tobie